tests/installation/zdup.pm

```diff
@@ -65,7 +65,7 @@ sub run() {
     }
     else {
         #SUSEMIRROR not set, zdup from ftp source for online migration
-        if (check_var('TEST', "migration_zdup_online_sle12_ga")) {
+        if (get_var('TEST') =~ /migration_zdup_online_sle12_ga/) {
```

Comment:
Because every migration testcase will have fullupdate and minimalupdate in their names, or shorten one like mu (minimalupdate) and fu (fullupdate)